src/Monitor/Monitor/MonitorCmdletBase.cs

```diff
@@ -86,14 +86,22 @@ public override void ExecuteCmdlet()
             string reasonPhrase = null;
             string message = null;
             string exName = null;
-
+            string warningEnvironmentParam = "SuppressAzurePowerShellBreakingChangeWarnings";
```

Comment:
https://github.com/Azure/azure-powershell-common/blob/b58ba4053f9cbd2861813e6dc3bf4800713d1a7f/src/Common/CustomAttributes/BreakingChangeAttributeHelper.cs#L31

Should use this one.

Reply:
Got it,thanks.